### ISSUE TITLE
Breaking command line after quiting sbt console for Scala 2.11/2.12

### ISSUE THREAD

**Issue by seratch** [coursier/coursier#378]
I found enabling coursier sbt plugin breaks terminal inputs after quiting `sbt console`. The problem doesn't happen for Scala 2.10. See the actual behavior below:

http://showterm.io/08ced1f75303375f4464f

Thank you for creating coursier. coursier is life-changing! I hope this issue will be fixed soon.

**Comment by alexarchambault** [comment]
That was quickly discussed some time ago on gitter IIRC. @paulp suggested to do something along the lines of [what sbt-extras does](https://github.com/paulp/sbt-extras/blob/9a362aee2ac3750a5b1abe3a76ebbe8744027a42/sbt#L47-L58). This kind of things should be added to `TermDisplay`.

**Comment by kzys** [comment]
Hmm, does coursier write something to console even before downloading artifacts?

**Comment by alexarchambault** [comment]
@kzys If progress bars are enabled, it does [this](https://github.com/alexarchambault/coursier/blob/master/cache/src/main/scala/coursier/TermDisplay.scala#L384).

**Comment by kzys** [comment]
Interesting. Doing the equivalent from Ruby doesn't break console. Maybe sbt (or jline) does some more?

```
aug12:~ kazuyoshi$ echo $SHELL
/bin/bash
aug12:~ kazuyoshi$ echo $TERM
xterm-256color
aug12:~ kazuyoshi$ ruby -e 'print "\e[2K"'
aug12:~ kazuyoshi$ 
aug12:~ kazuyoshi$ 
aug12:~ kazuyoshi$ 
```

**Comment by dwijnand** [comment]
As a workaround, you might find that `stty sane` fixes it (at least it does sometimes for me on macOS).

**Comment by larsrh** [comment]
@dwijnand On Linux, I just use `reset`.

**Comment by alexarchambault** [comment]
That should be fixed (since `1.0.0-RC1` at least). Feel free to re-open if you still have problems with that.

**Comment by jeffrey-aguilera** [comment]
Still have the problem with `1.0.0-RC3`.  Only occurs if I enter the `console`.
